modules/grpc/otel/filterx/object-otel-scope.cpp

```diff
@@ -99,6 +99,52 @@ Scope::get_subscript(FilterXObject *key)
     }
 }
 
+bool
+Scope::unset_key(FilterXObject *key)
+{
```

Comment:
the commit summary refers to Resource whereas this commit changes the "scope" implementation.

Reply:
thanks, fixed